src/H5Ppublic.h

```diff
@@ -420,6 +420,11 @@ typedef enum H5D_selection_io_mode_t {
 } H5D_selection_io_mode_t;
 //! <!--[H5D_selection_io_mode_t_snip] -->
 
+/* Actual selection I/O modes for H5Pget_actual_selection_io_mode() property */
+#define H5D_SCALAR_IO    0x1 /* Scalar (or legacy MPIO) I/O was performed */
+#define H5D_VECTOR_IO    0x2 /* Vector I/O was performed */
+#define H5D_SELECTION_IO 0x4 /* Selection I/O was performed */
```

Comment:
The default is 0, which isn't a value here. What does it mean when the function returns zero? That's not indicated in the Doxygen for the new API call, but it's tested for in the test.

Reply:
This is a bitflags type parameter. 0 indicates no flags set (no raw data I/O peformed). Flags should only be set when I/O is performed so it always starts out as 0.

Reply:
We could make this clearer in the documentation but I don't think this needs to be changed. It's worth keeping in mind that the "default" is not really a default in the same way as other properties since this property is not set by the user.